### PR TITLE
feat(input): support input type="range" for @value

### PR DESCRIPTION
**Pull request by gpolychronis**
- Parse the value into integer, so that it's ready for use.
- Number.isInteger is not supported by IE. How should we handle this?

-  Added an example, accessible at http://localhost:5000/examples/range/

### CODE REVIEW

File: src/iv/inputs.ts [hunk 1]
```diff
@@ -42,6 +42,13 @@ export const value = decorator(Value, ($api: Value) => {
         let v: any;
         if (inputType === "text" || inputType === "number") {
             v = elt[VALUE];
+        } else if (inputType === "range") {
+            const value = elt[VALUE];
+            try {
+                v = parseInt(value);
```
Comment (divdavem) [inline src/iv/inputs.ts]: Note that `parseInt` does not raise an exception when the value is not a number! It returns `NaN`.
So what's the purpose of this try...catch block?

Comment (gpolychronis) [reply src/iv/inputs.ts]: Fixed.

File: src/doc/examples/range/range.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+}
+
+// @@extract: template
+export const range = template(`($:RangeCtl) => {
+    <div class="range">
+        <input type="range"
+            min={$.min}
+            max={$.max}
+            @value={=$.value}
+            @onmouseup={e=>$.select()}
```
Comment (b-laporte) [inline src/doc/examples/range/range.ts]: you can remove the "e" argument in this case: @onmouseup={=>$.select()}

File: src/doc/examples/range/range.ts [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+            @ontouchend={e=>$.select()}/>
```
Comment (b-laporte) [inline src/doc/examples/range/range.ts]: same here

File: src/doc/examples/range/range.ts [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+        </div>
+    </>
+}`, RangeCtl);
+
+const main = template(`(message="", $) => {
+    const onSelect = (selection: number) => {
+        $.message="Selected value: " + selection;
+    };
+    <*range placeholder=30 min=0 max=100
+        @onselect={e=>onSelect(e.data)}
```
Comment (b-laporte) [inline src/doc/examples/range/range.ts]: I would rather have all the event handler inline (and remove the local onSelect function) - to keep things as simple as possible:
@onselect={e=>$.message="Selected value: " + e.data}

**Pull request merged**